docs/root/version_history/current.rst

```diff
@@ -23,6 +23,7 @@ Minor Behavior Changes
 * kill_request: enable a way to configure kill header name in KillRequest proto.
 * memory: enable new tcmalloc with restartable sequences for aarch64 builds.
 * mongo proxy metrics: swapped network connection remote and local closed counters previously set reversed (`cx_destroy_local_with_active_rq` and `cx_destroy_remote_with_active_rq`).
+* outlier detection: added :ref:`max_ejection_time <envoy_v3_api_field_config.cluster.v3.OutlierDetection.max_ejection_time>` to limit ejection time growth when node stays unhealthy for extended period of time.
```

Comment:
Can you make it more clear that the old behavior was unlimited growth and the new default is 5 minutes, etc.? Thank you.

/wait

Reply:
Expanded the release note.